Updated main.js added validation code

thivatm opened the pull request:
Added validation for not to use space in Nome field!

thivatm committed:
Updated main.js

Added validation for not to use space in Nome field!

robertoduessmann commented:
Hi @thivatm , I really thank you by your interest and for contribution :tada: 
But this validation already created by @mrdedavid in https://github.com/robertoduessmann/polling/pull/17 / https://github.com/robertoduessmann/polling/pull/21

Feel free to take a look on project issues or improve another part of app :grinning:

robertoduessmann commented:
Hi @thivatm , I really thank you by your interest and for contribution :tada: 
But this validation already created by @mrdedavid in https://github.com/robertoduessmann/polling/pull/17 / https://github.com/robertoduessmann/polling/pull/21

Maybe you can improve the validation to show a friendly message to user :)

Or, if you think better can take a look in another project issue or improve another part of app :grinning:

thivatm committed:
Again updated main.js 

User will get an alert if he uses blank space in the Question name field. I have updated my previous validation code!

robertoduessmann commented:
Merged, thanks for contribution @thivatm :tada:
